honeybadgermpc/hbavss.py

```diff
@@ -229,57 +232,144 @@ def __exit__(self, typ, value, traceback):
         for task in self.tasks:
             task.cancel()
 
-    async def _process_avss_msg(self, avss_id, dealer_id, dispersal_msg):
+    async def _process_avss_msg(self, avss_id, dealer_id, rbc_msg, avid):
```

Comment:
Consider splitting this method, this is an extremely long method.

Reply:
Yep